examples/memcache/main.tf

```diff
@@ -14,9 +14,6 @@
  * limitations under the License.
  */
 
-provider "google-beta" {
```

Comment:
We should pin this to a version and keep this

Reply:
Done.